package.json

```diff
@@ -59,6 +59,5 @@
     "unified": "^11.0.3",
     "vitest": "^1.4.0",
     "zod": "^3.22.4"
-  },
-  "packageManager": "pnpm@8.15.4+sha256.cea6d0bdf2de3a0549582da3983c70c92ffc577ff4410cbf190817ddc35137c2"
```

Comment:
Will investigate later if this affects anything. The app is currently building without `pnpm`.